queries.sql

```diff
@@ -1,3 +1,34 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+/* List the name of all animals born between 2016 and 2019. */
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-12-31';
+
+/* List the name of all animals that are neutered and have less than 3 escape attempts. */
+
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
```

Comment:
**Required Changes**
- [ ] Let us place use `true` instead of `t` where query is **TRUE**. This makes our query more clear.

Reply:
Thank's @oxenprogrammer  for the review! Change's have been made 😄